test/unit/nfts/ERC1155MaxSupplyMintable.t.sol

```diff
@@ -58,19 +58,6 @@ contract UnitTestERC1155MaxSupplyMintable is BaseTest {
         assertEq(nft.maxTokenSupply(tokenId + 1), supplyCap);
     }
 
-    function testSetTokenSupplyCapAtDeploymentWithRoleSucceds() public {
```

Comment:
Any reason why this unit test was removed?

Reply:
yes the function they were testing was removed as it was a `DEPLOYER` only function.